pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/SourceConfig.java

```diff
@@ -58,7 +58,6 @@
     @isImplementationOfClass(implementsClass = SerDe.class)
     private String serdeClassName;
 
-    @isImplementationOfClass(implementsClass = Schema.class)
```

Comment:
why do we need to remove this.  Shouldn't we check if the class implements Schema?

Reply:
We allow inbuilt things like avro, json, etc, which make this not the right check

Reply:
I see, can we implement a validator to check for those?

Reply:
Validating schemas is a little complicated becuase we need the types to do full validation. Thus their validation class cannot just be annotations, but actually involve more logic after we extract the types. Thus in the ValidatorImpls class, validation of schemas is done without using annotations